### ISSUE TITLE
train

### ISSUE THREAD

**Issue by CarTonK** [FanChiMao/HWMNet#15]
The loss will become very big during training, I haven't changed the code, and I can't run your results.

**Comment by xxk2002** [comment]
I encountered the same problem, may I ask if you have resolved it now?

**Comment by FanChiMao** [comment]
Which dataset are you using for training?  
Please note that the hyperparameters differ between datasets, as mentioned in the README.
If you are training on a custom dataset and encountering exploding gradients,  
try reducing the learning rate (such as 1e-5).

For reference:  
```python
 -------------------------------------------------
 LOL dataset:
 Training patches: 4850 (485 x 10)
 Validation: 15
 Initial learning rate: 5e-5
 Final learning rate: 1e-5
 Training epochs: 300

 MIT-5K dataset:
 Training patches: 36000 (4500 x 8)
 Validation: 500
 Initial learning rate: 1e-4
 Final learning rate: 1e-6
 Training epochs: 100
 -------------------------------------------------
```

**Comment by xxk2002** [comment]
Thank you very much for your reply, the problem has now been solved!
